src/panels/lovelace/cards/hui-light-card.ts

```diff
@@ -0,0 +1,274 @@
+        </div>
+      </ha-card>
+    `;
+  }
+
+  protected shouldUpdate(changedProps: PropertyValues) {
+    if (changedProps.get("hass")) {
+      return (
+        (changedProps.get("hass") as any).states[this._config!.entity] !==
+        this.hass!.states[this._config!.entity]
+      );
+    }
+    return (changedProps as unknown) as boolean;
+  }
+
+  protected firstUpdated() {
+    const brightness = this.hass!.states[this._config!.entity].attributes
+      .brightness;
+    jQuery("#light", this.shadowRoot).roundSlider({
+      ...lightConfig,
+      change: (value) => this._setBrightness(value),
+      drag: (value) => this._dragEvent(value),
+      start: () => this._showBrightness(),
+      stop: () => this._hideBrightness(),
+    });
+    this.shadowRoot!.querySelector(".brightness")!.innerHTML =
+      (Math.round((brightness / 254) * 100) || 0) + "%";
+  }
+
+  protected updated() {
+    const attrs = this.hass!.states[this._config!.entity].attributes;
+
+    jQuery("#light", this.shadowRoot).roundSlider({
+      value: Math.round((attrs.brightness / 254) * 100) || 0,
+    });
+  }
+
+  private renderStyle() {
+    return html`
+    ${roundSliderStyle}
+    <style>
+      :host {
+        display: block;
+      }
+      ha-card {
+        position: relative;
+        overflow: hidden;
+      }
+      #tooltip {
+        position: absolute;
+        top: 0;
+        left: 0;
+        right: 0;
+        height: 100%;
+        text-align: center;
+        z-index: 15;
+      }
+      .icon-state {
+        display: block;
+        margin: auto;
+        width: 100%;
+        height: 100%;
+        transform: translate(0,25%);
+      }
+      #light {
+        margin: 0 auto;
+        padding-top: 16px;
+        padding-bottom: 16px;
+      }
+      #light .rs-bar.rs-transition.rs-first, .rs-bar.rs-transition.rs-second{
+        z-index: 20 !important;
+      }
+      #light .rs-range-color  {
+        background-color: var(--primary-color);
+      }
+      #light .rs-path-color  {
+          background-color: #d6d6d6;
+      }
+      #light .rs-handle  {
+          background-color: #FFF;
+          padding: 7px;
+          border: 2px solid #d6d6d6;
+      }
+      #light .rs-handle.rs-focus  {
+          border-color:var(--primary-color);
+      }
+      #light .rs-handle:after  {
+          border-color: var(--primary-color);
+          background-color: var(--primary-color);
+      }
+      #light .rs-border  {
+        border-color: transparent;
+      }
+      ha-icon {
+        margin: auto;
+        width: 76px;
+        height: 76px;
+        color: var(--paper-item-icon-color, #44739e);
+        cursor: pointer;
+      }
+      ha-icon[data-state=on] {
+        color: var(--paper-item-icon-active-color, #FDD835);
+      }
+      ha-icon[data-state=unavailable] {
+        color: var(--state-icon-unavailable-color);
+      }
+      .name {
+        padding-top: 40px;
+        font-size: 1.2rem;
+      }
+      .brightness {
```

Comment:
The text is not always very visible as the icon color, which is the background of brightness, changes.

Reply:
Added a stroke to it but not sure how material design esque this is

Reply:
it's a good start, we can iterate on it.